skrl/memories/torch/base.py

```diff
@@ -261,13 +261,36 @@ def add_samples(self, **tensors: torch.Tensor) -> None:
 
         # dimensions and shapes of the tensors (assume all tensors have the dimensions of the first tensor)
         tmp = tensors.get("states", tensors[next(iter(tensors))])  # ask for states first
+        if isinstance(tmp, dict):
+            # Handle custom case (integrating with ManiSkill).
+            assert set(tmp.keys()) == {"state", "rgb"}
+            tmp = tmp["state"]
         dim, shape = tmp.ndim, tmp.shape
 
         # multi environment (number of environments equals num_envs)
         if dim == 2 and shape[0] == self.num_envs:
             for name, tensor in tensors.items():
                 if name in self.tensors:
-                    self.tensors[name][self.memory_index].copy_(tensor)
+                    if isinstance(tensor, dict):
+                        # Handle custom case (integrating with ManiSkill).
+                        assert set(tensor.keys()) == {"state", "rgb"}
+                        self.tensors[name][self.memory_index].copy_(
+                            torch.cat(
+                                [
+                                    tensor["state"].view(tensor["state"].shape[0], -1),
```

Comment:
Can we export out which one or if we want to run them both, state and rgb, into the config? It would be nice to be able to choose easily, state vs visual vs state-visual. Thanks.